tests/runtime/engine/runner.py

```diff
@@ -136,6 +138,31 @@ def __get_bpffeature():
         bpffeature["get_func_ip"] = output.find("get_func_ip: yes") != -1
         return bpffeature
 
+    
+    @staticmethod
+    def __wait_for_children(parent_pid, timeout, ps_format, condition):
+         with open(os.devnull, 'w') as dn:
+            waited=0
+            while waited <= timeout:
+                run_cmd = ["ps", "--ppid", str(parent_pid), "--no-headers", "-o", ps_format]
+                children = subprocess.run(run_cmd,
+                                          check=False,
+                                          stdout=subprocess.PIPE,
+                                          stderr=subprocess.PIPE)
+                if children.returncode == 0 and children.stdout:
+                    lines = [line.decode('utf-8') for line in children.stdout.splitlines()]
+                    if (condition(lines)):
+                        return True
+                else:
+                    print(f"__wait_for_children error: {children.stderr}")
```

Comment:
```suggestion
                    print(fail(f"__wait_for_children error: {children.stderr}"))
```

Reply:
This is just for debugging. I don't think we want it to have the fail colors.